tests/testthat/_snaps/verb-joins.md

```diff
@@ -40,6 +40,37 @@
       LEFT JOIN `foo2`.`df` AS `df_RHS`
         ON (`df_LHS`.`x` = `df_RHS`.`x`)
 
+# alias truncates long table names at database limit
+
+    Code
+      self_join2 %>% remote_query()
+    Output
+      <SQL> SELECT `a012345678901234567890123456789012345678901234567890123456789012_L`.*
+      FROM `a012345678901234567890123456789012345678901234567890123456789012` AS `a012345678901234567890123456789012345678901234567890123456789012_L`
+      LEFT JOIN `a012345678901234567890123456789012345678901234567890123456789012` AS `a012345678901234567890123456789012345678901234567890123456789012_R`
+        ON (
+          `a012345678901234567890123456789012345678901234567890123456789012_L`.`x` = `a012345678901234567890123456789012345678901234567890123456789012_R`.`x` AND
+          `a012345678901234567890123456789012345678901234567890123456789012_L`.`y` = `a012345678901234567890123456789012345678901234567890123456789012_R`.`y`
+        )
+
+---
+
+    Code
+      self_join3 %>% remote_query()
+    Output
+      <SQL> SELECT
+        `a012345678901234567890123456789012345678901234567890123456789012...1`.`x` AS `x`,
```

Comment:
That table name is 68 characters long?

Reply:
After using `method = "both.sides"` in `abbreviate()`, this case is no longer an issue. This is a good example where `method = "left.keep"` preserves too much of the LHS at the expense of having unnecessary length.